docs/shared/tutorials/epic-web-app.md

```diff
@@ -0,0 +1,495 @@
+---
+title: 'Epic Web App Tutorial'
+description: In this tutorial you'll add Nx to an existing NPM workspaces repo
+---
+
+# Epic Web App Tutorial
+
+In this tutorial, you'll learn how to add Nx to an app created with the [Epic Stack](https://www.epicweb.dev/epic-stack). You'll see how Nx can provide immediate value with very little configuration and then you can gradually enable more features.
```

Comment:
I'd make this section a little more specific to the tutorial, like focusing on the *why* you want to add Nx.

In the list I'd drop "add Nx to the repo with a single command" as in this context that doesn't really help, but focus on the

- adding task caching
- making sure tasks run in the correct order (task pipeline)
- use the Nx Playwright plugin to enhance the Playwright experience